### PR TITLE
feat(SwitchButton): add new type && more style

### PR DESCRIPTION
**Pull request by ZengBeauty**
## Description

Add some more Props && fit more style.

## Type of change

Please select the relevant option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by ZengBeauty** [comment]
企业微信20210602-182644@2x

**Comment by ShinyLeee** [comment]
@ZengBeauty 合并下 [#64](https://github.com/tuya/tuya-panel-kit/pull/64) example 多语言规划化

File: src/components/switch-button/switch-button.js [hunk 1]
```diff
@@ -94,11 +95,31 @@ export default class SwitchButton extends React.PureComponent {
...
+     */
+    iconSize: PropTypes.number,
+    /**
+     * 滑块上的图标颜色
+     */
+    iconColor: PropTypes.string,
+    /**
+     * Switch 开关类型，默认不传，特殊情况传值 thumbMore
+     */
+    switchType: PropTypes.string,
```
Comment (ShinyLeee) [inline src/components/switch-button/switch-button.js:118]: 可枚举值的类型

**Review by ShinyLeee — CHANGES_REQUESTED** [review]
附下 typing

**Comment by ZengBeauty** [comment]
typing 详见 https://github.com/tuya/DefinitelyTyped/commit/a6803228d812a5dc663358d435a8dc262f1355a7

**Review by ShinyLeee — APPROVED** [review]

**Review by ShinyLeee — CHANGES_REQUESTED** [review]
单测覆盖率下降了 10%，新增的属性需覆盖全

File: example/src/i18n/strings.js [hunk 2]
```diff
@@ -457,5 +473,11 @@ module.exports = {
     unitText_custom_color_art: 'Customize the color of each value of UnitText(ART version)',
 
     theme_global_type: 'Global type',
-  },
```
Comment (youngjuning) [inline example/src/i18n/strings.js:460]: @ZengBeauty 这里多删了一个小括号？

**Review by youngjuning — CHANGES_REQUESTED** [review]
加一个小括号！

**Pull request merged**